bsp/boards/MSP430F5438A/SConscript

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
The MSP430F5438A is a chip, not a board. Please rename this directory to the name of the board.